examples/training/classification/basic_imagenet_training.py

```diff
@@ -206,6 +213,9 @@ def augment(img, label):
 
 
 callbacks = [
+    callbacks.ReduceLROnPlateau(
```

Comment:
patience=20 will generalize better.  Augmentation is very stochastic.

Reply:
Done.